api/src/models/tag.model.ts

```diff
@@ -0,0 +1,13 @@
+import { model, Schema } from 'mongoose';
+import { ITag } from 'shared';
```

Comment:
Avoid I prefix for interfaces
https://github.com/microsoft/TypeScript-Handbook/issues/121